src/gluonts/model/forecast.py

```diff
@@ -350,6 +350,8 @@ def __init__(
         ), "start_date should be a pandas Period object"
         self.start_date = start_date
 
+        if freq is None:
+            freq = self.start_date.freqstr
```

Comment:
Is the idea to still accept `freq` as input for backward compatibility? Because since it's actually redundant, it may lead to confusion. Should we deprecate it?

Reply:
I believe its not needed... In all my PR I try to make sure I do not break anything but yes we can slowly deprecate the freq arg here

Reply:
so i believe the Predictor also uses it but I did not chase up why the Predictor needs the freq, its most probably also not needed... but I didn't want to break anything...

Reply:
I think I'm fine with removing it, do tests pass though?

Reply:
hmm i can check...

Reply:
@jaheba yes I believe i can make things work if I remove freq... should i do it?

Reply:
Let's break things!